crowdsec-docs/unversioned/getting_started/post_installation/acquisition.mdx

```diff
@@ -0,0 +1,252 @@
+
+<details>
+
+<summary>Example output</summary>
+
+```bash title="See highlighted lines at bottom"
+type: collections
+name: crowdsecurity/iptables
+file_name: iptables.yaml
+description: 'iptables support : logs and port-scans detection scenarios'
+author: crowdsecurity
+path: collections/crowdsecurity/iptables.yaml
+version: "0.2"
+parsers:
+- crowdsecurity/iptables-logs
+scenarios:
+- crowdsecurity/iptables-scan-multi_ports
+contexts:
+- crowdsecurity/firewall_base
+local_path: /etc/crowdsec/collections/iptables.yaml
+local_version: "0.2"
+local_hash: d59e4198c2b88cccb6f9da9f9375348a1835d7f5a933d7452693ad0321ef1282
+installed: true
+downloaded: true
+uptodate: true
+tainted: false
+belongs_to_collections:
+- crowdsecurity/unifi
+local: false
+
+# highlight-start
+Current metrics:
+
+- (Parser) crowdsecurity/iptables-logs:
+╭────────────────────────┬──────┬────────┬──────────╮
+│        Parsers         │ Hits │ Parsed │ Unparsed │
+├────────────────────────┼──────┼────────┼──────────┤
+│ file:/var/log/kern.log │ 45   │ 45     │ 0        │
+├────────────────────────┼──────┼────────┼──────────┤
+│ file:/var/log/syslog   │ 45   │ 45     │ 0        │
+╰────────────────────────┴──────┴────────┴──────────╯
+
+- (Scenario) crowdsecurity/iptables-scan-multi_ports:
+╭───────────────┬───────────┬──────────────┬────────┬─────────╮
+│ Current Count │ Overflows │ Instantiated │ Poured │ Expired │
+├───────────────┼───────────┼──────────────┼────────┼─────────┤
+│ 0             │ 0         │ 38           │ 38     │ 38      │
+╰───────────────┴───────────┴──────────────┴────────┴─────────╯
+# highlight-end
+```
+
+</details>
+
+Within the output if the bottom section is blank or the parsed count is `0` then you may need to perform additional checks on the configuration. Follow the [how to setup a new acquisition](#how-to-setup-a-new-acquisition) section which will walk you through the steps to add a new acquisition.
+
+## What services are supported?
+
+To find a list of supported services, you can [browse the hub online](https://hub.crowdsec.net/) or use the `cscli` command line tool.
+
+```bash
+cscli collections list --all
+```
+
+:::tip
+You can filter the output by piping the output using command line tools such as `grep` or `awk`.
+:::
+
+## Next steps?
+
+If you already see all services you want covered by the previous commands then you can [head back to the post installation guide.](/getting_started/next_steps.md)
+
+However, if there is a service you wish to cover that is already covered by the [Hub](https://hub.crowdsec.net/) then follow the next steps.
+
+## How to setup a new acquisition
+
+Once you have found service you want to add, you can use the `cscli` command line tool to add the collection.
```

Comment:
Should we put the tip about the fact that the hub page will usually tell you which files to monitor earlier, like here?